registry/emailTemplates.xml

```diff
@@ -69,4 +69,5 @@
 	<email key="STATISTICS_REPORT_NOTIFICATION" subject="emails.statisticsReportNotification.subject" body="emails.statisticsReportNotification.body"/>
 	<email key="ANNOUNCEMENT" subject="emails.announcement.subject" body="emails.announcement.body"/>
 	<email key="EDITORIAL_REMINDER" can_disable="0" from_role_id="16" to_role_id="1048576" subject="emails.editorialReminder.subject" description="emails.announcement.description" body="emails.editorialReminder.body"/>
+	<email key="DISCUSSION_NOTIFICATION" subject="discussion.notification.subject" body="discussion.notification.body"/>
```

Comment:
I think we need to add a line to the `upgrade.xml` file to install this template on upgrade for each application.

Reply:
Yep